week-3/Homework/mandatory/6-lane-names.js

```diff
@@ -4,10 +4,11 @@
   Write a function that will return all street names which contain 'Lane' in their name.
 */
 
-function getLanes() {
-
+function getLanes(arr) {
+   let newArr = arr.filter(e => e.includes ("Lane"))
```

Comment:
This works, but is slightly more code than it needs to be. You don't really need the `newArr` variable here - it isn't really providing any value, and so it may be a bit clearer to instead write:
`return arr.filter(e => e.includes("Lane"));`

Reply:
Sabia please write element instead of e would be more clear